docs/features.md

```diff
@@ -0,0 +1,64 @@
+# Features
+
+This document gives an overview over Alacritty's features beyond its terminal
+emulation capabilities. To get a list with supported control sequences take a
+look at [Alacritty's escape sequence support](./escape_support.md).
+
+## Vi Mode
+
+The vi mode allows moving around Alacritty's viewport and scrollback using the
+keyboard. It also serves as a jump-off point for other features like search and
+opening URLs with the keyboard. By default you can launch it using
+<kbd>Ctrl</kbd> <kbd>Shift</kbd> <kbd>Space</kbd>.
```

Comment:
There's a way to add a `plus` sign in between for it to look more homogeneous.

Reply:
I'm not sure what I like more tbh. I feel like without the plus formats a little nicer, the `+` looks a little out of place:

<kbd>Ctrl</kbd>+<kbd>c</kbd>
<kbd>Ctrl</kbd> <kbd>c</kbd>

Reply:
I do like it in a way `Ctrl + c` tbqh, but it uses a bit different formatting, than `kbd`.

Reply:
I personally prefer <kbd>Ctrl</kbd> <kbd>c</kbd> out of all of them. It doesn't have an out of place looking connecting character, is still clear and makes it obvious that this is about a keybinding.